numpydoc/docscrape_sphinx.py

```diff
@@ -79,21 +79,74 @@ def _str_returns(self, name='Returns'):
                 out += ['']
         return out
 
-    def _str_param_list(self, name):
+    def _str_param_list(self, name, fake_autosummary=False):
         out = []
         if self[name]:
+            if fake_autosummary:
+                prefix = getattr(self, '_name', '')
+                if prefix:
+                    autosum_prefix = '~%s.' % prefix
+                    link_prefix = '%s.' % prefix
+                else:
+                    autosum_prefix = ''
+                    link_prefix = ''
+                autosum = []
+
             out += self._str_field_list(name)
             out += ['']
             for param, param_type, desc in self[name]:
+                param = param.strip()
+
+                display_param = '**%s**' % param
+
+                if fake_autosummary:
+                    param_obj = getattr(self._obj, param, None)
+                    if not (callable(param_obj)
+                            or isinstance(param_obj, property)
+                            or inspect.isgetsetdescriptor(param_obj)):
+                        param_obj = None
+                    obj_doc = pydoc.getdoc(param_obj)
+
+                    if param_obj and (obj_doc or not desc):
```

Comment:
it turns out the "or not desc" logic here wasn't working because `desc` is being provided as `['']` when there is no description line. Which means no one depends on this logic, which means I'm getting rid of it because it's weird (unless you can think of a use-case for linking to an autogen page where the attribute has no docstring).